src/cljs/athens/views.cljs

```diff
@@ -62,33 +63,34 @@
 
 (defn main
   []
-  (let [loading    (rf/subscribe [:loading?])
-        modal      (rf/subscribe [:modal])]
+  (let [loading (rf/subscribe [:loading?])
+        modal (rf/subscribe [:modal])]
     (fn []
-      [:<>
-       [alert]
-       (let [{:keys [msg type]} @(rf/subscribe [:db/snack-msg])]
-         [m-snackbar
-          {:message msg
-           :open (boolean msg)}
-          [:span
-           {:style {:background-color (case type
-                                        :success "green"
-                                        "red")
-                    :padding "10px 20px"
-                    :color "white"}}
-           msg]])
-       [athena-component]
-       (cond
-         (and @loading @modal) [filesystem/window]
-
-         @loading [initial-spinner-component]
-
-         :else [:<>
-                (when @modal [filesystem/window])
-                [:div (stylefy/use-style app-wrapper-style)
-                 [app-toolbar/app-toolbar]
-                 [left-sidebar/left-sidebar]
-                 [pages/view]
-                 [right-sidebar/right-sidebar]
-                 [devtool-component]]])])))
+      [hotkeys/hotkeys
+       [:<>
```

Comment:
Now we don't need React Fragment anymore, since `hotkeys/hotkeys` is new root node.
This would require `hotkeys` to accept `[& children]` instead of `[children]`.

Reply:
Ohh so that's how you can have `multiple` child node passed. Sounds great.